src/main/java/com/synopsys/integration/alert/workflow/startup/AlertStartupInitializer.java

```diff
@@ -146,12 +154,10 @@ private void updateConfigurationFields(final String descriptorName, final boolea
         throws AlertDatabaseConstraintException {
         if (!configurationModels.isEmpty()) {
             if (!foundConfigurationModels.isEmpty()) {
-                if (overwriteCurrentConfig) {
-                    final ConfigurationModel configurationModel = foundConfigurationModels.get(0);
-                    logger.info("  Overwriting configuration values with environment for descriptor.");
-                    final Collection<ConfigurationFieldModel> updatedFields = updateAction(descriptorName, configurationModels);
-                    fieldConfigurationAccessor.updateConfiguration(configurationModel.getConfigurationId(), updatedFields);
-                }
+                final ConfigurationModel foundModel = foundConfigurationModels.get(0);
+                logger.info("  Overwriting configuration values with environment for descriptor.");
```

Comment:
Why do these logs have a space at the beginning?

Reply:
indentation to nest them so it's a little easier to read the server startup logs.